data.py

```diff
@@ -40,12 +40,13 @@ def __init__(self, is_validation=False, validation_percentage=.1, data_dir=Path(
         self.sentences = [self.text2code(s) for s in sents]
 
         num_validation = math.floor(len(self.strokes) * validation_percentage)
```

Comment:
The changes look good but while looking at this, I think that we should rename `validation_percentage` to `val_proportion` since we're not using the number as a percentage.